src/Components/Chart.svelte

```diff
@@ -65,7 +66,15 @@
 		const xs = innerData.map((p) => p.x);
 		const ys = innerData.map((p) => p.y);
 
-		return isValidSelection(f1, f2) ? getPearsonCorrelation(xs, ys) : null;
+		if (isValidSelection(f1, f2)) {
+			const xValid = isQuant(xs)
+			const yValid = isQuant(ys)
+			errorMessage = !xValid ? "First parameter has non-numeric values" : errorMessage
+			errorMessage = !yValid ? "Second parameter has non-numeric values" : errorMessage
```

Comment:
It could be that both parameters have non-numeric values. Perhaps append to the string instead of overwriting it to show that both have errors